### ISSUE TITLE
Orthonormality test is too strict

### ISSUE THREAD

**Issue by Nikdwal** [JuliaManifolds/Manifolds.jl#379]
I'm implementing the manifold of  tensors of fixed multilinear rank and the test code seems to be much too strict.
I have a `DefaultOrthonormalBasis`, and the normalisation is checked in `src/tests_general.jl` using the following line:
https://github.com/JuliaManifolds/Manifolds.jl/blob/7e90c455804311f1220bcb52878fa5abf86e0abb/src/tests/tests_general.jl#L540

This test gives the following output:
```
Tucker: **Test Failed** at .../Manifolds.jl/src/tests/tests_general.jl:540
  Expression: ≈(norm(M, p, Xs_invs[i]), 1, atol = find_eps(p))
   Evaluated: 0.9999999999999993 ≈ 1 (atol=2.220446049250313e-16)
```

It seems appropriate to change the value of `atol` to something like `√find_eps(p)`, as is done three lines down.

**Comment by mateuszbaran** [comment]
Yes, sure, `sqrt(find_eps(p))` in that place makes sense.

**Comment by kellertuer** [comment]
I second that, sure, reduce the tolerance, that seems reasonable. Just do it on your branch.

And cool that you work on a new manifold for `Manifolds.jl`!
If you would like some feedback on your current state of implementation (haven't looked at your branch yet) feel free to open a PR already early on.
I hope the way how to implement a manifold is understandble; if not, also open the PR and we can help along the way.

**Comment by mateuszbaran** [comment]
Fixed in #380  .
